src/vs/workbench/electron-browser/main.contribution.ts

```diff
@@ -163,23 +163,34 @@ MenuRegistry.appendMenuItem(MenuId.MenubarFileMenu, {
 	order: 2
 });
 
-MenuRegistry.appendMenuItem(MenuId.MenubarFileMenu, {
-	group: '2_open',
-	command: {
-		id: OpenFileAction.ID,
-		title: nls.localize({ key: 'miOpenFile', comment: ['&& denotes a mnemonic'] }, "&&Open File...")
-	},
-	order: 1
-});
+if (!isMacintosh) {
```

Comment:
Can we use `when` clauses instead of the ugly `if` brackets,

Reply:
I would like to do that. I tried but I didn't see platform context keys, but I may have missed them. Are you aware?

Reply:
I checked and there is none at the moment.
But I would add it somewhere in this [file](https://github.com/Microsoft/vscode/blob/1616ea11935857a077657756eb957be36b7fe756/src/vs/workbench/electron-browser/workbench.ts#L647)
I would call it `platform` and it would be `windows|linux|macintosh`

But we do not have to do it for this PR, we can tackle later. Would you like me to create an issue so we do not forget about it?
fyi @bpasero

Reply:
@isidorn @sbatten this just landed via https://github.com/Microsoft/vscode/pull/54894

Reply:
https://github.com/Microsoft/vscode/issues/56274